source/locales/es/LC_MESSAGES/the_need_for.po

```diff
@@ -0,0 +1,473 @@
+msgstr ""
+
+#: ../../source/the_need_for.rst:59
+msgid "Final Ecosystem Services:"
+msgstr ""
+
+#: ../../source/the_need_for.rst:61
+msgid "Forest Carbon Edge Effect"
+msgstr ""
+
+#: ../../source/the_need_for.rst:63
+msgid "Carbon Storage and Sequestration"
+msgstr ""
+
+#: ../../source/the_need_for.rst:65
+msgid "Coastal Blue Carbon"
+msgstr ""
+
+#: ../../source/the_need_for.rst:67
+msgid "Annual Water Yield"
+msgstr ""
+
+#: ../../source/the_need_for.rst:69
+msgid "Nutrient Delivery Ratio"
+msgstr ""
+
+#: ../../source/the_need_for.rst:71
+msgid "Sediment Delivery Ratio"
+msgstr ""
+
+#: ../../source/the_need_for.rst:73
+msgid "Unobstructed Views: Scenic Quality Provision"
+msgstr ""
+
+#: ../../source/the_need_for.rst:75
+msgid "Visitation: Recreation and Tourism"
+msgstr ""
+
+#: ../../source/the_need_for.rst:77
+msgid "Wave Energy Production"
+msgstr ""
+
+#: ../../source/the_need_for.rst:79
+msgid "Offshore Wind Energy Production"
+msgstr ""
+
+#: ../../source/the_need_for.rst:81
+msgid "Marine Finfish Aquacultural Production"
+msgstr ""
+
+#: ../../source/the_need_for.rst:83
+msgid "Fisheries"
+msgstr ""
+
+#: ../../source/the_need_for.rst:85
+msgid "Crop Production"
+msgstr ""
+
+#: ../../source/the_need_for.rst:87
+msgid "Seasonal Water Yield"
+msgstr ""
+
+#: ../../source/the_need_for.rst:90
+msgid "Tools to Facilitate Ecosystem Service Analyses:"
+msgstr ""
+
+#: ../../source/the_need_for.rst:92
+msgid "Overlap Analysis"
+msgstr ""
+
+#: ../../source/the_need_for.rst:94
+msgid "Coastal Vulnerability"
+msgstr ""
+
+#: ../../source/the_need_for.rst:96
+msgid "InVEST GLOBIO"
+msgstr ""
+
+#: ../../source/the_need_for.rst:99
+msgid "Supporting Tools:"
+msgstr ""
+
+#: ../../source/the_need_for.rst:101
+msgid "RouteDEM"
+msgstr ""
+
+#: ../../source/the_need_for.rst:103
+msgid "DelineateIT"
+msgstr ""
+
+#: ../../source/the_need_for.rst:105
+msgid "Scenario Generator"
+msgstr ""
+
+#: ../../source/the_need_for.rst:107
+msgid "Scenario Generator: Proximity Based"
+msgstr ""
+
+#: ../../source/the_need_for.rst:110
+msgid "Using InVEST to Inform Decisions"
+msgstr ""
+
+#: ../../source/the_need_for.rst:112
+msgid ""
+"Information about changes in ecosystem services is most likely to make a "
+"difference when questions are driven by decision-makers and stakeholders,"
+" rather than by scientists and analysts. We have found that InVEST is "
+"most effective when used within a decision-making process. The Natural "
+"Capital Project has used InVEST in over 60 countries worldwide. See the "
+"Where We Work section of the NatCap website "
+"(https://naturalcapitalproject.stanford.edu/impact/where-we-work) for the"
+" latest map and description of our projects. Through our experience "
+"applying InVEST and helping to shape decisions, we have seen how the "
+"InVEST tool fits within the larger context of a natural capital approach."
+msgstr ""
+
+#: ../../source/the_need_for.rst:114
+#, python-format
```

Comment:
Out of curiosity, what's the `, python-format` here?

Reply:
Oh interesting, I hadn't noticed that. The comment lines starting with `#,` are flags which have special meaning to gettext programs. The `python-format` flag indicates that the message is a python format string (clearly incorrect here). I suspect it's because of the `%` in the string. Though, it's strange that the sphinx gettext utility isn't correcting for that.

Reply:
Pretty sure this is a bug in the sphinx gettext builder.